builders/build/eth-api/dev-env/foundry.md

```diff
@@ -185,7 +223,7 @@ Foundry包括cast，一个用于执行以太坊RPC调用的CLI。
 0x000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000074d79546f6b656e00000000000000000000000000000000000000000000000000
 ```
 
-此数据非可读，但您可以使用cast将其转换成您想要的格式。在这种情况下，数据是文本，因此您可以将其转换为ascii字符以查看“My Token”：
+此数据非可读，但您可以使用cast将其转换成您想要的格式。在这种情况下，数据是文本，因此您可以将其转换为ASCII字符以查看“My Token”：
 
 ![Foundry Contract View](/images/builders/build/eth-api/dev-env/foundry/foundry-3.webp)
```

Comment:
This image is removed and is replaced by:

--8<-- 'code/builders/build/eth-api/dev-env/foundry/terminal/cast.md'